src/app/components/ContactPlaceholder.js

```diff
@@ -134,34 +137,32 @@ const ContactPlaceholder = ({
 
     if (contactGroupID) {
         const { Name } = contactGroups.find(({ ID }) => ID === contactGroupID);
-        const handleClick = () => createModal(<ContactGroupModal contactGroupID={contactGroupID} />);
+        const total = contacts.filter(({ LabelIDs = [] }) => LabelIDs.includes(contactGroupID)).length;
+        const totalContactsText = (
+            <span className="color-primary bold">
+                {total === 1 ? c('Info').t`1 contact` : c('Info').t`${total} contacts`}
```

Comment:
Why not using `ngettext` here?
[Documentation](https://ttag.js.org/docs/ngettext.html)

Reply:
It's because of the color. Part of the text has color, so I need to translate a structure: `string` + `html` + `string`. In those cases I need to use `jt` to wrap the html inside a variable. The fact that that variable changes with plurals is just a side effect. Even if it didn't change, I would need to use `jt` anyway. If it were just a string changing with plurals, then I could use `ngettext`

Reply:
By the way, there's some inconsistency with designs. Here I'm using a bold blue color for the number of contacts, but in the ContactPlaceholder (the sentence saying "you have **n contacts** in your address book") we're using bold black. I asked @matstaz about it